### ISSUE TITLE
popover to render component 

### ISSUE THREAD

**Issue by Sunil6591** [kamranahmedse/driver.js#201]
Can we pass component (HTML Element) for popover to render? Currently it accepts only `description`. e.g.
```
driver.highlight({
      element: 'some_element_id',
      popover: {
        title: 'some title',
        description: 'This is description',
        position: 'top',
        component: <button /> // Either description or component can be passed on
      }
    });
```

**Comment by wayder315** [comment]
You can pass HTMLElement to 'element' property instead of query selector;

**Comment by kamranahmedse** [comment]
`description` does accept HTML so you should be able to do that.

Also we just released [v1.0](https://github.com/kamranahmedse/driver.js/releases/tag/1.0.3) of driver.js which has been rewritten from the ground up. Please have a [look at the docs](https://driverjs.com/) and do let me know if you face any issues.
